treasurer/accounting.csv

```diff
@@ -1,76 +1 @@
16-09-02,,54.95,Pub,Social night drinks,Hacky social night (Maddy cash),ch012,Yes
16-09-02,,55.98,Snacks,Snacks,,ch013,Yes
16-09-09,,125,Misc,CUPE Instructor honorarium,For Ricardo,ch008,Yes
16-09-15,,125,Misc,CUPE Instructor honorarium,For Samar,ch009,Yes
16-09-19,,88.75,Pub,Drinks for Fall term Hacky hour,(Luke cash),ch014,Yes
16-09-21,527.8,,SWC,SWC workshop,Registration fees,,
16-09-30,,2.5,BankFee,BMO Maintanence fee,,,
16-10-11,,48.92,Snacks,Snacks,For Luke,ch015,Yes
16-10-31,,2.5,BankFee,BMO Maintanence fee,,,
16-11-21,,125,Misc,CUPE Instructor honorarium,For Jeff,ch010,Yes
16-11-30,,2.5,BankFee,BMO Maintanence fee,,,
16-12-30,,2.5,BankFee,BMO Maintanence fee,,,
17-01-31,,2.5,BankFee,BMO Maintanence fee,,,
17-02-07,,25.06,Snacks,Snacks,For Luke,ch017,Yes
17-02-10,837.2,,SWC,"SWC workshop, Feb. 2-3, 2017",Registration fees,,
17-02-13,,409.63,SWC,Coffee and tea for SWC workshop (Feb 3),For Innis Cafe,ch018,Yes
17-02-23,,13.48,Misc,Supplies for SWC workshop (Feb 3),For Maddy,ch019,Yes
17-02-28,,2.5,BankFee,BMO Maintanence fee,,,
17-03-17,,128.5,Misc,Laptop stickers,For Lina,ch020,Yes
17-03-24,,6.76,Snacks,Snacks (with PR 106 for $48.56),For Luke,ch022,Yes
17-03-24,,41.8,Misc,Adapter for HDMI/VGA (with PR 106 for $48.56),For Luke,ch022,Yes
17-03-24,,58.95,Misc,Drinks and snacks for Hack day,For Luke,ch016,Yes
17-03-24,,31.97,Snacks,Snacks for sessions,For Luke,ch025,Yes
17-03-31,,2.5,BankFee,BMO Maintanence fee,,,
17-03-31,83.45,,Misc,From Maddy for Hack Day,,,
17-04-06,,9,Misc,Snacks for Hack day,For Maddy,ch024,Yes
17-04-11,,76.44,SWC,Snacks for SWC (Feb 2017),For Lindsay,ch021,Yes
17-04-24,,32.06,Snacks,Snacks for sessions,For Luke,ch026,Yes
17-04-24,,5.98,Snacks,Tea for sessions,For Luke,ch026,Yes
17-04-24,,33.89,Misc,Kettle for heating up water,For Luke,ch026,Yes
17-04-28,,15.5,Misc,Drinks for Hack day,For Lina,ch023,Yes
17-04-28,,2.5,BankFee,BMO Maintanence fee,,,
17-05-08,,44,Misc,Drinks for Winter term and election social night,For Joel,ch028,Yes
17-05-31,,2.5,BankFee,BMO Maintanence fee,,,
17-06-23,,28.3,Snacks,Snacks for coding sessions,For Luke,ch030,Yes
17-06-30,,2.5,BankFee,BMO Maintanence fee,,,
17-07-19,,27.6,Misc,Drinks for Winter term and election social night,For Elliott,ch027,Yes
17-07-31,,2.5,BankFee,BMO Maintanence fee,,,
17-08-31,,2.5,BankFee,BMO Maintanence fee,,,
17-09-22,,34.52,Snacks,Snacks for sessions,For Luke,ch031,Yes
17-09-22,,162.05,Misc,Drinks and apps for Fall 2017 social,For Luke,ch033,Yes
17-09-29,,2.5,BankFee,BMO Maintanence fee,,,
17-10-13,,22.59,Supplies,Ethernet for laptop to projector connection,For Joel,ch032,Yes
17-10-31,,2.5,BankFee,BMO Maintanence fee,,,
17-11-03,,32.3,Snacks,Snacks for sessions,For Luke,ch034,Yes
17-11-30,,2.5,BankFee,BMO Maintanence fee,,,
17-12-13,,80.25,Pub,Drinks and snacks for Winter 2017 Hacky Hour,For Luke,ch035,Yes
17-12-29,,2.5,BankFee,BMO Maintanence fee,,,
18-02-20,,100,Gift,Starbucks Gift Cards,,ch036,
18-02-20,,50.15,SWC,SWC supplies,,ch037,
```

Comment:
Is something up with the line endings here?

Reply:
There are missing cells in the .csv file because they are not related to cheques.